### PR TITLE
shl FilterByOrientationBias tag doc, flesh out example command, introduce tool

### PR DESCRIPTION
**Pull request by sooheelee**
Please see <https://github.com/broadinstitute/gatk/issues/3030> for the errors I encountered when running this tool. I have not been able to run the tool successfully and so ask what are we missing in the documentation that will help users get this tool running.

Otherwise, I recommend labeling this tool experimental as well as in BETA.

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/exome/FilterByOrientationBias.java [hunk 1]
```diff
@@ -25,9 +26,44 @@
 import java.util.*;
 import java.util.stream.Collectors;
 
+/**
+ * Additionally filter Mutect2 somatic variant calls for sequence-context dependent artifacts.
+ *
+ * <p>
+ *     This tool is complementary to {@link org.broadinstitute.hellbender.tools.walkers.mutect.FilterMutectCalls}.
+ *     The tool requires the pre-adapter detailed metrics calculated by Picard CollectSequencingArtifactMetrics.
+ *     Specify the sequence context(s) to consider for filtering with the --artifactModes argument.
```
Comment (davidbenjamin) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/FilterByOrientationBias.java]: >Specify the sequence contexts(). . .

This isn't quite right.  Rather it specifies what types of substitutions eg G --> T to consider.  For every substitution, the Picard detailed metrics measure orientation bias at all 3-base contexts, but the user can't control that.  Note also that if you specify a substitution the tool filters both it and its reverse complement eg G --> T *and* C --> A.  Only one substitution in a reverse complement pair should be specified.

**Comment by davidbenjamin** [comment]
@sooheelee I think the experimental label is fine for now.  Lee is still evaluating it.

File: src/main/java/org/broadinstitute/hellbender/tools/exome/FilterByOrientationBias.java (hunk 1 above)
Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/FilterByOrientationBias.java]: Incorporated

**Comment by sooheelee** [comment]
Lee is looking for someone else to evaluate it @davidbenjamin. In the meanwhile, because we have the BETA/Experimental label on this tool, it should be ok to release.

**Comment by sooheelee** [comment]
I will let someone else be the decider on this one. Perhaps @LeeTL1220 or @droazen?

**Comment by sooheelee** [comment]
@davidbenjamin I've incorporated an experimental/BETA label. Am I missing something from your review?

**Comment by davidbenjamin** [comment]
Thumbs up!

**Pull request merged**